css/signin.css

```
@@ -107,7 +106,7 @@ input:focus {
 
 .social-login {
   display: flex;
-  width: 100%;
+  width:100%;
```

여기 말고도 전체적으로 확인 부탁드립니다! 

```suggestion
  width: 100%;
```